src/qe/mbp/mbp_term_graph.h

```diff
@@ -23,6 +29,7 @@ Module Name:
 #include "util/plugin_manager.h"
 #include "qe/mbp/mbp_solve_plugin.h"
 #include "model/model.h"
+#include "util/vector.h"
```

Comment:
this seems wrong. there is no vector used in this header file. move include to where it is needed.